examples/montage/index.html

```diff
@@ -1,26 +1,18 @@
-<!doctype html>
-<html lang="en" data-framework="montage" id="montage-todomvc">
-	<head>
-		<meta charset="utf-8">
-		<title>Montage • TodoMVC</title>
+<!DOCTYPE html><html lang="en" data-framework="montage" id="montage-todomvc"><head>
+        <meta charset="utf-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <title>Montage • TodoMVC</title>
 
-		<link rel="stylesheet" href="bower_components/todomvc-common/base.css">
-		<link rel="stylesheet" href="assets/app.css">
-	</head>
-	<body>
-		<div data-montage-id="todo-container" id="todo-container"></div>
+        <link rel="stylesheet" href="packages/todomvc-common@13305a5/base.css" />
+        <link rel="stylesheet" href="packages/todomvc-app-css@c556082/index.css" />
+        <link rel="stylesheet" href="assets/app.css" />
+    </head>
+    <body>
+        <div data-montage-id="todo-container" id="todo-container"></div>
 
-		<script src="bower_components/todomvc-common/base.js"></script>
-		<script src="index.html.bundle-0.js" data-montage-location="packages/montage@6364dae/" data-promise-location="packages/q@2847ee2/" data-montage-hash="6364dae" data-promise-hash="2847ee2" data-application-hash="6607c26"></script>
-		<script type="text/montage-serialization">
-			{
-				"owner": {
-					"prototype": "montage/ui/loader.reel",
-					"properties": {
-						"element": {"#": "todo-container"}
-					}
-				}
-			}
-		</script>
-	</body>
-</html>
+        <script src="packages/todomvc-common@13305a5/base.js"></script>
+        <script src="index.html.bundle-0.js" data-montage-location="packages/montage@23c6465/" data-promise-location="packages/q@833279d/" data-montage-hash="23c6465" data-promise-hash="833279d" data-application-hash="ad457b1"></script>
+        <script type="text/montage-serialization">{"owner":{"prototype":"montage/ui/loader.reel","properties":{"element":{"#":"todo-container"}}}}</script>
+    
+
+</body></html>
```

Comment:
What would you like to see here? Montage is based on components and the index.html won't look quite like https://github.com/tastejs/todomvc-app-template/blob/06a7873ca21a449bcb16036fdb84e6b06afed840/index.html in source form, but it does when the app runs